apps/bare-expo/metro.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const { createMetroConfiguration } = require('expo-yarn-workspaces');
+
+const blacklist = require('metro-config/src/defaults/blacklist');
+const config = createMetroConfiguration(__dirname);
+
+const blacklistRE = () =>
```

Comment:
Instead of redefining the blacklist here, it seems like we should update the config that e-y-w creates. That said, if we already define the blacklist in the `resolver` configuration it seems like it should therefore filter out the blacklisted files from ever getting transformed too.